pkg/test/framework/features/whitelist.txt

```diff
@@ -0,0 +1,1679 @@
+// This file contains all test cases known to exist before 6/8/2020
+// It will be used to allow old test cases to lack feature labels,
+// while failing on new tests that don't use labels.
+authn_policy,TestAuthnPolicy
+authz_deny,TestAuthzDeny
+authz_ingress,TestAuthzIngress
+authz_jwt,TestAuthzJWT
+authz_tcp,TestAuthzTCP
+cert_mtls_test,TestCertMtls
+cert_provision_prometheus,TestPrometheusCert
+client_tracing_test,TestClientTracing
+cni,TestCNIReachability
+cni,TestCNIReachability/global-mtls-on
+cni,TestCNIReachability/global-mtls-on/a->grpc://a:grpc
```

Comment:
maybe we should only add the top level? like TestCNIReachability here. just to make it a bit smaller